wrapper/test/tests.py

```diff
@@ -111,12 +131,14 @@ def create_limited_path(self, executables):
 
         return path_dir + ":/bin"
 
-    def run_script(self, args, download_url=default_download_url, path=os.environ["PATH"]):
+    def run_script(self, args, download_url=default_download_url, path=os.environ["PATH"], quiet_download="undefined"):
         env = {
             "BATECT_CACHE_DIR": self.cache_dir,
             "BATECT_DOWNLOAD_URL": download_url,
             "PATH": path
         }
+        if quiet_download != "undefined":
```

Comment:
This feels a little bit more idiomatic to me:

```suggestion
        if quiet_download != None:
```